.github/workflows/ci_dockers.yml

```diff
@@ -43,7 +43,7 @@ jobs:
             PYTORCH_VERSION=${{ matrix.pytorch_version }}
           file: dockers/release/Dockerfile
           push: false
-        timeout-minutes: 50
+        timeout-minutes: 70  # FIXME
```

Comment:
Reviewers: Reinstalling horovod requires extra time, and 70 seems very reasonable as seen in https://github.com/PyTorchLightning/pytorch-lightning/runs/5590124760?check_suite_focus=true.